std/file.d

```diff
@@ -5198,6 +5200,12 @@ if (__traits(compiles, { DirEntry entry; bool _ = pred(entry); }))
     sort(result);
 
     assert(equal(files, result));
+
+    import std.algorithm : endsWith;
+    auto result2 = dirEntries!((scope ref DirEntry entry) => entry.name.endsWith("Hello World"))(dir, SpanMode.shallow).map!((return a) => a.name.normalize()).array();
+    import std.stdio;
+    writeln(result2);
```

Comment:
I hope this is temporary.

Reply:
Yes.

Reply:
We're going to resolve this comment for now to move the PR along.